### DISCUSSION TITLE
Verschieben von Apps

### DISCUSSION THREAD

**Discussion by Petung** [d4rken-org/sdmaid-se#738] [Ideas]
Wird es eine Funktion geben, Apps auf SD-Karte zu verschieben?

**Comment by MarkyMarkDE** [comment]
> Wird es eine Funktion geben, Apps auf SD-Karte zu verschieben?

soweit ich weiß, gibt es dieses Feature seit Android 5(?) nicht mehr, Google will das nicht mehr und hat es ersatzlos aus Android gestrichen.
Ohne die nötige Schnittstelle in der API wird das nicht mehr zu ermöglichen sein.
